package.json

```diff
@@ -31,6 +31,7 @@
     "react-dnd-html5-backend": "^5.0.1",
     "react-dom": "^16.5.2",
     "shapeup": "0.2.5",
+    "svg-sprite": "^1.3.6",
```

Comment:
We actually left this out intentionally so that we didn't install it unless you needed to build the svg's

Reply:
Ok, will remove...